### PR TITLE
Blank project option

### LINKED ISSUE #261: Blank project option
The second wizard page (from New Project) should have the option "_Blank project \n Draw your graphs_". When clicking on this button, the program should go straight to the Project Window.

A blank .g6 file must be created with the name and location chosen by the user in the first wizard window.

The tree must be loaded in the location chosen by the user (from the .g6 file).

**_Warning:_** the program must allow opening a blank graph (empty line in the file)

image

### PR DESCRIPTION
**Pull request by rosaigor138**
Closes #261

### CODE REVIEW

File: source/controller/controller.py [hunk 1]
```diff
@@ -103,7 +105,13 @@ def finish_filter(self):
 
     def start_filter(self):
```
Comment (atilaajones) [inline source/controller/controller.py:106]: rename `finish_filter` to `start_project`
rename `start_filter` to `finish_wizard`

File: source/controller/controller.py [hunk 2]
```diff
@@ -103,7 +105,13 @@ def finish_filter(self):
...
         update_project_store()
-        self.filter_controller.start_filter()
+        if project_information_store.method == 'blank':
+            graph = nx.Graph()
+            create_g6_file(project_information_store.project_location+"/"+project_information_store.project_name+".g6", nx.to_graph6_bytes(graph, header=False).decode('utf-8'))
```
Comment (atilaajones) [inline source/controller/controller.py:110]: Format long lines.

**Review by atilaajones — CHANGES_REQUESTED** [review]

File: source/controller/controller.py [hunk 3]
```diff
@@ -103,7 +105,13 @@ def finish_filter(self):
 
     def start_filter(self):
         update_project_store()
-        self.filter_controller.start_filter()
+        if project_information_store.method == 'blank':
+            graph = nx.Graph()
```
[the remark this answers is not in the document]
Comment (rosaigor138) [reply source/controller/controller.py:109]: Can be done one method who does all that in utils, but I dont think it's necessary

[the remark this answers is not in the document]
Comment (atilaajones) [reply source/controller/controller.py:109]: It's not really be necessary, it is for organization. Even the operations that @fsoupimenta did, I order to remove ` Operations` from `Controller` and move function to `Store`.

I think that in the future, a simple adaptation can be made and use the `EmptyGraph(NewGraphStore)` function itself in the `new_graph_store`.

**Review by atilaajones — APPROVED** [review]

**Pull request merged**